src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -165,7 +165,11 @@ public virtual void Process (LinkContext context)
 			IFlowAnnotationSource annotationSource = new AttributeFlowAnnotationSource ();
 			if (_context.AttributeDefinitions != null && _context.AttributeDefinitions.Count > 0) {
 				annotationSource = new AggregateFlowAnnotationSource (
-					_context.AttributeDefinitions.Select (s => new JsonFlowAnnotationSource (_context, s))
+					_context.AttributeDefinitions.Select (s => {
+						var xmlAnnotations = new XmlFlowAnnotationSource (_context, s);
+						xmlAnnotations.ParseXml ();
```

Comment:
Nit: Cleaner design would be to parse the file name to the `ParseXml` file instead...